kong.yml

```diff
@@ -212,4 +213,32 @@ nginx: |
       # Do not remove, additional configuration placeholder for some plugins
       # {{additional_configuration}}
     }
+
+    server {
+      listen {{consumer_api_port}};
+
+      location / {
+        default_type application/json;
+        content_by_lua '
+          ngx.header["Access-Control-Allow-Origin"] = "*"
```

Comment:
This is too much code duplication. If this port is to be implemented, it needs to abstract what already exists for port 8001 and reuse it.